models/inline_strconv_parse_test.go

```diff
@@ -0,0 +1,69 @@
+package models_test
+
+import (
+	"strconv"
+	"testing"
+	"testing/quick"
+
+	"github.com/influxdata/influxdb/models"
+)
+
+func TestParseIntBytesEquivalenceFuzz(t *testing.T) {
+	f := func(b []byte, base int, bitSize int) bool {
```

Comment:
I updated the PR with two more tests in line with what you suggest here (although I do not explicitly use reflection).